tests/unit/ArtifactTypeListener.spec.js

```diff
@@ -0,0 +1,65 @@
+import ArtifactTypeListener from "src/listener/ArtifactTypeListener.js";
+import metadata from "../data/metadata.json";
+let parsedRule = {
+   value: {
+      derived_from: {
+         value: "tosca.artifact.Root",
+         type: "string",
+      },
+      description: {
+         value: "Java Archive artifact type",
+         type: "string",
+      },
+      mime_type: {
+         value: "application/java-archive",
+         type: "string",
+      },
+      // properties: {
+      //    value: {
+      //       creator: {
+      //          tosca: {
+      //             name: "creator",
+      //             type: {
+      //                constraints: null,
+      //                description:
+      //                   "Vendor of the java implementation on which the jar is based",
+      //                type: "string",
+      //             },
+      //          },
+      //       },
+      //       id: {
+      //          tosca: {
+      //             name: "id",
+      //             type: {
+      //                constraints: null,
+      //                description: "Unique identifier of the artifact",
+      //                type: "string",
+      //             },
+      //          },
+      //       },
+      //    },
+      // },
```

Comment:
What is the purpose of this comment? Maybe it should be deleted

Reply:
It's a draft i prepared for a more complex example of the test. I will delete it for now.